index.html

```diff
@@ -31,7 +31,7 @@
 
 <p>SpotBugs is the spiritual successor of <a href="http://findbugs.sourceforge.net/">FindBugs</a>, carrying on from the point where it left off with support of its community.</p>
 
-<p>SpotBugs requires JRE (or JDK) 1.7.0 or later to run.  However, it can analyze programs compiled for any version of Java, from 1.0 to 1.8.</p>
+<p>SpotBugs requires JRE (or JDK) 1.8.0 or later to run.  However, it can analyze programs compiled for any version of Java, from 1.0 to 1.8.</p>
```

Comment:
we could take the chance to say it can analyze any java program from 1.0 to 1.9 actually